src/main/java/seedu/duke/command/EstimateCommand.java

```diff
@@ -0,0 +1,66 @@
+package seedu.duke.command;
+
+import seedu.duke.anime.AnimeData;
+import seedu.duke.exception.AniException;
+import seedu.duke.human.User;
+import seedu.duke.human.Workspace;
+import seedu.duke.storage.StorageManager;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static seedu.duke.logger.AniLogger.getAniLogger;
+
+public class EstimateCommand extends Command {
+    private static final int NO_WORDS_PER_HOUR_PROVIDED = -1;
+    private static final int MINUTES_PER_HOUR = 60;
+    private static final int[] AVERAGE_TRANSLATOR_WORDS_PER_HOUR = {400, 500, 600};
```

Comment:
Perhaps some meaning could be added to those numbers.

Reply:
Alright, I will add some comments to explain the numbers.